### PR TITLE
Enable SendReceive_VaryingLengthBuffers_Success on Windows run

### PR DESCRIPTION
**Pull request by caesar1995**
Have run the test 500 times on local Windows 10 machine. No failure. The previous stack trace #22845 is too old, cannot reflect which code path results in this failure.

Enable the test on Windows to see if it fails in CI. If it fails, I can investigate further.

Related: #23765

### CODE REVIEW

File: src/System.Net.WebSockets.Client/tests/testloop.bat [hunk 1]
```diff
@@ -0,0 +1,14 @@
+SETLOCAL EnableDelayedExpansion
```
Comment (stephentoub) [inline src/System.Net.WebSockets.Client/tests/testloop.bat]: Did you mean to add this file?

File: src/System.Net.WebSockets.Client/tests/testloop.bat [hunk 2]
```diff
@@ -0,0 +1,14 @@
...
+SETLOCAL ENABLEEXTENSIONS
+
+for /L %%x in (1,1,500) do (
+
+mkdir "D:/TestDiag/Run %%x"
```
Comment (davidsh) [inline src/System.Net.WebSockets.Client/tests/testloop.bat]: I don't recommend we hard code drive letters in these scripts.  Not everyone has a "D:" drive.  I don't for example.  I have my other drives set to letters like "S:" (enlistments) and "W:" (work stuff)

File: src/System.Net.WebSockets.Client/tests/testloop.bat (hunk 1 above)
Comment (caesar1995) [reply src/System.Net.WebSockets.Client/tests/testloop.bat]: No... It's removed in the new commit.

File: src/System.Net.WebSockets.Client/tests/testloop.bat [hunk 3]
```diff
@@ -0,0 +1,14 @@
...
+msbuild /t:rebuildandtest > "D:/TestDiag/Run %%x/output.log"
+
+IF NOT ERRORLEVEL 1 (
+rmdir "D:/TestDiag/Run %%x/" /s /q
```
Comment (davidsh) [inline src/System.Net.WebSockets.Client/tests/testloop.bat]: same comment as above about drive letters.

**Comment by rmkerr** [comment]
It looks like the only failures in windows were on 8.1 and not 10. I think it might be worth trying to recreate the situation more closely by running the script in a VM on 8.1.

**Comment by davidsh** [comment]
>It looks like the only failures in windows were on 8.1 and not 10. I think it might be worth trying to recreate the situation more closely by running the script in a VM on 8.1.

Weren't those Win 8.1 failures a while ago and when we had the WinHTTP implementation for Windows?

Now, we use the managed sockets implementation for both Linux and Windows. So, maybe getting failures on Win 8.1 won't be possible (or the same type of failure) as before.

**Comment by rmkerr** [comment]
I hadn't considered the switch from WinHTTP to managed sockets. Since the code under test has changed, this looks good to me.

**Comment by caesar1995** [comment]
@dotnet-bot test Outerloop Windows x64 Debug Build
@dotnet-bot test Outerloop Windows x86 Debug Build
@dotnet-bot test Outerloop Windows x64 Release Build
@dotnet-bot test Outerloop Windows x86 Release Build

**Comment by caesar1995** [comment]
> Now, we use the managed sockets implementation for both Linux and Windows.

Yes, I found the related PR here: #26429. If there is no failure reported after this PR get merged, we can consider close #23765.

**Comment by caesar1995** [comment]
Unrelated failure: System.Net.WebSockets.Client.Tests.CancelTest/CloseAsync_Cancel_Success(server: ws://corefx-net.cloudapp.net/WebSocket/EchoWebSocket.ashx)

I can open a new issue and investigate it. (Doesn't look like this issue #22904)

**Comment by stephentoub** [comment]
@dotnet-bot test Outerloop Windows x64 Debug Build
@dotnet-bot test Outerloop Windows x86 Debug Build
@dotnet-bot test Outerloop Windows x64 Release Build
@dotnet-bot test Outerloop Windows x86 Release Build
@dotnet-bot test OSX x64 Debug Build please

**Comment by stephentoub** [comment]
@dotnet-bot test Windows x86 Release Build please

**Comment by caesar1995** [comment]
Disable `System.Net.WebSockets.Client.Tests.CancelTest/CloseAsync_Cancel_Success` (#27211)

@dotnet-bot test Outerloop Windows x64 Debug Build
@dotnet-bot test Outerloop Windows x86 Debug Build
@dotnet-bot test Outerloop Windows x64 Release Build
@dotnet-bot test Outerloop Windows x86 Release Build

**Comment by caesar1995** [comment]
Disable `System.Net.Http.Functional.Tests.HttpClientHandlerTest/GetAsync_AllowAutoRedirectTrue_RetainsOriginalFragmentIfAppropriate` (#27217)

@dotnet-bot test Outerloop Windows x64 Debug Build
@dotnet-bot test Outerloop Windows x86 Debug Build
@dotnet-bot test Outerloop Windows x64 Release Build
@dotnet-bot test Outerloop Windows x86 Release Build

**Comment by caesar1995** [comment]
@dotnet-bot test Outerloop Windows x86 Debug Build
@dotnet-bot test Outerloop Windows x64 Release Build
@dotnet-bot test Outerloop Windows x86 Release Build

**Comment by rmkerr** [comment]
Have you gotten a chance to look and see when those tests started failing?

**Pull request merged**